cms/djangoapps/contentstore/permissions.py

```diff
@@ -0,0 +1,24 @@
+"""
+Permission definitions for the contentstore djangoapp
+"""
+
+from bridgekeeper import perms, rules
+from common.djangoapps.student.roles import CourseCreatorRole
+from lms.djangoapps.courseware.rules import HasRolesRule
+
+# Is the user active (and their email verified)?
```

Comment:
Minor: it may be a bit better to not phrase the comment as a question.

Reply:
As the name itself are descriptive, so removing comments.